mappings/net/minecraft/client/MinecraftClient.mapping

```diff
@@ -7,7 +7,7 @@ CLASS net/minecraft/class_310 net/minecraft/client/MinecraftClient
 	FIELD field_1689 framebuffer Lnet/minecraft/class_276;
 	FIELD field_1690 options Lnet/minecraft/class_315;
 	FIELD field_1692 targetedEntity Lnet/minecraft/class_1297;
-	FIELD field_1693 is64Bit Z
+	FIELD field_1693 64Bit Z
```

Comment:
Is that a valid field name?

Reply:
Not valid, failed to build:
`Caused by: java.lang.IllegalArgumentException: not a valid name: 64Bit`

Reply:
Maybe `x64`?